File: test/org/stringtemplate/v4/test/Playground.java
package org.stringtemplate.v4.test;

import org.stringtemplate.v4.*;

public class Playground {
	public static void main(String[] args) {
		ErrorBufferAllErrors errors = new ErrorBufferAllErrors();
		STGroup g = new STGroupFile("/tmp/g.stg");
		g.setListener(errors);
		ST t = g.getInstanceOf("u");
		if ( t!=null ) System.out.println(t.render());
		System.err.println("errors: "+errors);
	}
}

File: objc/main/tests/output/Playground.m
#import "Playground.h"

@implementation Playground

+ (void) main:(NSArray *)args {
  ErrorBufferAllErrors * errors = [[[ErrorBufferAllErrors alloc] init] autorelease];
  STGroup * g = [[[STGroupFile alloc] init:@"/tmp/g.stg"] autorelease];
  [g setListener:errors];
  ST * t = [g getInstanceOf:@"u"];
  if (t != nil)
    [System.out println:[t render]];
  [System.err println:[@"errors: " stringByAppendingString:errors]];
}

@end
